app/views/komponenty/article.njk

```diff
@@ -0,0 +1,370 @@
+                                </svg>
+                            </span>
+                        </a>
+                    </div>
+                    <div class="govuk-grid-row" id="zivotne-situacie">
+                        {{ idskRelatedContent({
+                        customWidthClass : "full",
+                        heading: "Súvisiace témy",
+                        items: [
+                            {
+                            title: "Počet subjektov aktuálne",
+                            link: "#"
+                            },
+                            {
+                            title: "Odklad povinnej aktivácie",
+                            link: "#"
+                            }
+                        ]
+                    }) }}
+                    </div>
+                </div>
+            </div>
+        {% endblock %}
+{% block footer %}
+            {{ idskFooterExtended({
+        characterLength : 350,
+        upButtonEnabled : "true",
+        logo: "footer-extended-logo.svg",
+        subtitles: [
+        {
+        item:"Pomoc", 
+        href:"#"}, 
+        {
+        item:"Cookies",
+        href:"#"
+        }, 
+        {
+        item:"Kontakty",
+        href:"#"
+        }, 
+        { 
+        item:"Mapa Stránky",
+        href:"#"
+        }, 
+        {
+        item:"RSS",
+        href:"#"
+        }, 
+        {
+        item:"Vyhlásenie o prístupnosti",
+        href:"#"
+        }
+        ],
+        dropdownItems: [
+        {
+        value: "published",
+        text: "Validačná chyba"
+        },
+        {
+        value: "updated",
+        text: "Gramatická chyba",
+        selected: true
+        },
+        {
+        value: "views",
+        text: "Chyba responzivity"
+        },
+        {
+        value: "comments",
+        text: "Chyba iná"
+        }
+        ]
+        }) }}
+        {% endblock %}
+{% block scripts %}
+  <script type="text/javascript">
+  showNews = function (e) {
+    var $contentButton = document.getElementById('idsk-article__content__button');
+    var $content = document.getElementById('idsk-article__content__news');
+
+    if ($content.classList.contains('idsk-article__content__news--visible')){
+        $content.classList.remove('idsk-article__content__news--visible');
+        $contentButton.innerText = $contentButton.dataset.closed;
+    } else {
+        $content.classList.add('idsk-article__content__news--visible');
+        $contentButton.innerText = $contentButton.dataset.opened;
+    }
+}
+  </script>
+{% endblock %}
```

Comment:
ako tu mi to az tak nevadi, ale potom pri komponentoch bude treba si ten formatter nejak nastavit aby to tak nerozhadzovalo zle sa to cita potom

Reply:
Hej, nechapem preco to robi